docs/cedarling/cedarling-policy-store.md

```diff
@@ -155,94 +161,156 @@ This record contains the information needed to validate tokens from this issuer:
+  "claim_mapping": {  
+    "claim_name": {
+      "parser": "regex",
+      "type": "Acme::Email",
+      ...
+    },
+  }
 }
 ```
 
-- **trusted** : (Boolean) The type of token
-- **principal_id** : (String) The claim used to create the Cedar entity associated with this token.
-- **role_mapping** : (String, *optional*) The claim used to create a role for the token. The default value of `role_mapping` is `role`. The claim can be string or array of string.
+###### Fields
+
+- **type** : (*String*) Type of token (e.g., `id_token`, `userinfo`, `access`, `transaction`).
+- **user_id** : (*String*) Claim used as a unique identifier for the user (e.g., `email`, `sub`, `uid`).
+- **role_mapping** : (*String*) Mapping used to identify the user's role (e.g., `role`, `memberOf`).
+- **claim_mapping** : (*Object*) Definitions for parsing individual claims.
+- **claim_name**: (*Object*) Metadata for mapping the claim.
+- **parser** : (*String*) The parser to use (e.g., `regex`, `json`).
+- **type** : (*String*) Data type (e.g., `Acme::Email`).
+
+###### Non-Normative Example of Token Entity
 
-**Note**: Only one token should include the `role_mapping` field in the list of `token_metadata`.
+This example demonstrates how to parse claims in various formats using regex and json parsers.
+
+```json
+{
+  "user_id": "aud",
+  "role_mapping": "role",
+  "claim_mapping": {  
+    "email": {
+      "parser": "regex",
+      "type": "Acme::Email",
+      "regex_expression": "^(?P<UID>[^@]+)@(?P<DOMAIN>.+)$",
+      "UID": {"attr": "uid", "type": "string" },
+      "DOMAIN": {"attr": "domain", "type": "string" }
+    },
+    "profile": {
+      "parser": "regex",
+      "type": "Acme::Email",
+      "regex_expression": "(?x) ^(?P<SCHEME>[a-zA-Z][a-zA-Z0-9+.-]*):\/\/ (?P<HOST>[^\/:#?]+) (?::(?P<PORT>\d+))?  (?P<PATH>\/[^?#]*)? (?:\?(?P<QUERY>[^#]*))?  (?:#(?P<FRAGMENT>.*))?$",
+         "SCHEME": { "attr": "scheme", "type": "string" }, 
+         "HOST": { "attr": "host", "type": "string" }, 
+         "PORT": { "attr": "port", "type": "string" }, 
+         "PATH": { "attr": "path", "type": "string" }, 
+         "QUERY": { "attr": "query", "type": "string" }, 
+         "FRAGMENT": { "attr": "fragment", "type": "string" }
+    },
+    "dolphin": {
+      "parser": "json", 
+      "type": "Acme::Dolphin"
+    },
+  }
+}
+```
 
 ## Example Policy store
 
-Here is a non-normative example of a `cedarling_store.json` file:
+Below is a non-normative example of a `cedarling_store.json` file:
 
 ```json
 {
-    "cedar_version": "v4.0.0",
-    "policies": {
-        "840da5d85403f35ea76519ed1a18a33989f855bf1cf8": {
-            "description": "simple policy example",
-            "creation_date": "2024-09-20T17:22:39.996050",
-            "policy_content": "cedar_policy_encoded_in_base64"
-        }
+  "cedar_version": "v4.0.0",
+  "name": "Jans::store",
+  "description": "a simple policy example",
+  "policies": { 
+    "fbd921a51b...": {
```

Comment:
Please, left only one policy. Because `cedarling` support only one policy in store per file. And in `agama-lab` will do the same